codecov.yml

```diff
@@ -1,14 +1,9 @@
-ignore:
-  - src/Behat
-  - demos
-  - docs
-  - tools
 comment: false
 coverage:
   status:
     project:
       default:
         target: auto
-        threshold: 0.025
+        threshold: 0.015
```

Comment:
0.015% equals passes 1 uncovered line, but not 2

non-zero is needed to prevent rounding/random errors by Covecov